GETTING_STARTED.md

```diff
@@ -6,28 +6,9 @@ This is a general guide to setting up an Open Food Network **development environ
 
 Head to our wiki on [Learning Rails](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Learning-Rails) to find some good starting points.
 
-### Requirements
-
-The fastest way to make it work locally is to use Docker, you only need to setup git, see the [Docker setup guide](docker/README.md).
```

Comment:
I am in favour of keeping the Docker setup mention, it should still allow you to setup a local environment. The docker README does mention the limitations, and who knows maybe someone will come along and fix the docker setup !

Reply:
Good point. But the limitation is pretty limiting (not able to run system specs), so it's probably better to move to the bottom of the section, and mention that it has limitations.